src/main/java/Duke.java

```diff
@@ -1,10 +1,160 @@
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Yo Wassup, da name's\n" + logo);
+        System.out.println("Hi bro I'm Duke\n" + "Whatcha wanna do?");
+    }
+
+    /**
+     * Returns actions for lists based on user input.
+     */
+    public static void listActions() {
+        Scanner in = new Scanner(System.in);
+        String line;
+
+        do {
+            line = in.nextLine();
+            if (line.contains("done")){
+                updateTask(line,list);
+            }else if (line.equalsIgnoreCase("bye")){
+                printBye();
+                break;
+            }else if(line.equalsIgnoreCase("list")) {
+                printList(list);
+            }else if (line.contains("deadline")) {
+                createDeadline(line);
+                System.out.format("Now you have %s task%s in the list.\n", NUM_OF_TASKS,((NUM_OF_TASKS==1?"":"s")));
+            }else if (line.contains("event")) {
+                createEvent(line);
+                System.out.format("Now you have %s task%s in the list.\n", NUM_OF_TASKS,((NUM_OF_TASKS==1?"":"s")));
+            }else {
+                createToDo(line);
+                System.out.format("Now you have %s task%s in the list.\n", NUM_OF_TASKS,((NUM_OF_TASKS==1?"":"s")));
+            }
+        } while (!line.equalsIgnoreCase("bye"));
+
+    }
+
+    /**
+     * Adds new task to the list.
+     *
+     * @param t new Task
+     */
+    public static void addToList(Task t){
+        list[NUM_OF_TASKS] = t;
+        NUM_OF_TASKS++;
+        System.out.print("Got it. I've added this task:\n" + "[" + t.getTypeOfTask() + "]" + "[" + t.getStatusIcon() + "] "+
+                t.description);
+    }
+
+    /**
+     * Creates a new ToDo
+     *
+     * @param line User input
+     */
+    public static void createToDo(String line){
+        String description = line.substring(5);
+
+        ToDo toDo = new ToDo(description);
+        addToList(toDo);
+        System.out.println();
+    }
+
+    /**
+     * Creates a new Deadline
+     *
+     * @param line User input
+     */
+    public static void createDeadline(String line){
+        int startOfDateTime = line.indexOf("by");
+        String dateTime = line.substring(startOfDateTime+3);
+        String description = line.substring(9,startOfDateTime-1);
+
+        Deadline d = new Deadline(description,dateTime);
+        addToList(d);
+        System.out.println("(by: "+ d.getDateTime() + ")");
+    }
+
+    /**
+     * Creates a new Event
+     *
+     * @param line User input
+     */
+    public static void createEvent(String line){
+        int startOfDateTime = line.indexOf("at");
+        String dateTime = line.substring(startOfDateTime+3);
+        String description = line.substring(6,startOfDateTime-1);
+
+        Event e = new Event (description,dateTime);
+        addToList(e);
+        System.out.println("(at: "+ e.getDateTime() + ")");
+    }
```

Comment:
Very similar chunks of code (line 97 to 120), consider generalizing with a createTask method (also with todo if possible)?